src/Form/Control.php

```diff
@@ -38,12 +38,21 @@ class Control
      */
     public static function control(array $schema, string $type, $value): array
     {
-        if (!in_array($type, Control::CONTROL_TYPES)) {
+        $handlers = Configure::read('Control.handlers', []);
```

Comment:
Why not have a configuration indexed by property type?
This way we could have a simpler code:

```php
$handler = Configure::read('Control.handler.' . $type);
if (!empty($handler)) {
        return call_user_func_array([$handler['class'], $handler['method']], [$value, $schema]);
}
```
resto of the method code is unchanged